client/main.go

```diff
@@ -161,3 +161,12 @@ func TinkWorkflowClient() (workflow.WorkflowServiceClient, error) {
 	}
 	return workflow.NewWorkflowServiceClient(conn), nil
 }
+
+// TinkTemplateClient creates a new template client
+func TinkTemplateClient() (template.TemplateServiceClient, error) {
+	conn, err := GetConnection()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return template.NewTemplateServiceClient(conn), nil
+}
```

Comment:
Lets get rid of this from this PR please. boots does not need template access so lets wait until an actual need comes up for this.